### DISCUSSION TITLE
CSS @keyframe animations omitted from prod build

### DISCUSSION THREAD

**Discussion by heswell** [facebook/docusaurus#9060] [Q&A]
I've created some react components, which render svg and which we are using in our docusaurus site. These depend on css keyframe animations. In dev, everything is fine. The CSS imported by my components is included in the CSS bundle which is deployed to build when I run a prod build, but all the @keyframes declarations are missing, (so of course none of our animations work). 

I assume whatever css processor webpack is configured with is doing this - how can I get at this to try and fix it ?

I've tried hacking the css into a global css file included using clientModules with no success - any css except for @keyframe animations gets included, the @keyframe animations are stripped.

If I deploy them as a static asset, is there a way to add the link to include them to the index.html ?

**Comment by heswell** [comment]
Only solution I was able to get working. Take the css from my component and add it to a stylesheet in static assets. Add to stylesheets in config.

**Comment by slorber** [comment]
https://docusaurus.io/docs/next/cli#docusaurus-build-sitedir

Try:
- `--no-minify`build option
- `USE_SIMPLE_CSS_MINIFIER=true`

Outside of that it's impossible to you help more without a minimal repro that isolates the problem
